cypress/integration/delete-collection.ts

```diff
@@ -0,0 +1,24 @@
+describe("Let's delete from collection", () => {
+  it('Adds cards from search page', () => {
```

Comment:
```suggestion
  it('can add cards from search page to the collection', () => {
```

this should read like a sentence (starting with "it")